docs/react/faq.en-US.md

```diff
@@ -11,6 +11,17 @@ Here are the frequently asked questions about Ant Design and antd that you shoul
 
 There is currently no plan to add support for Sass/Stylus(etc.) style files, but using tools on Google you can easily convert the provided Less files to your desired style format.
 
+## Is there a difference between undefined and null in the controlled components of antd?
+
+**Yes. antd convention: `undefined` is uncontrolled, while `null` is an explicit empty value of controlled.**
```

Comment:
antd will treat `undefined` as uncontrolled but `null` as controlled component which mean empty value of it.